scripts/test

```diff
@@ -2,7 +2,7 @@
 set -e
 set -x
 
-if [ "$ARCH" = "mingw32" ] || [ "$ARCH" = "mingw64" ] || [ "$ARCH" = "arm32" ]; then
+if [ "$ARCH" = "mingw32" -o "$ARCH" = "mingw64" -o "$ARCH" = "arm32" ]; then
```

Comment:
It is late here... You probably want to move the 'if [ "$ARcH" = "" ]` test  up to before this line so that we can be sure that ARCH is set.

Reply:
Thank you! Yes, I moved it to up.